partners_api/partners_api_tests/uber_tests.cpp

```diff
@@ -176,6 +176,7 @@ UNIT_TEST(Uber_Smoke)
   vector<uber::Product> productsContainer;
   ms::LatLon const from(38.897724, -77.036531);
   ms::LatLon const to(38.862416, -76.883316);
+  uber::SetUberUrlForTesting("http://localhost:34568/partners");
```

Comment:
Instead of manual clear ath the end of the function I suggest you to use scope guards:
```c++
uber::UberAPI::SetUberUrlForTesting(...);
MY_SCOPE_GUARD(cleanup, []() {
  uber::UberAPI::SetUberUrlForTesting("");
});
```